registry/session.go

```diff
@@ -1,43 +1,26 @@
 package registry // import "github.com/docker/docker/registry"
 
 import (
-	"bytes"
-	"crypto/sha256"
-
 	// this is required for some certificates
 	_ "crypto/sha512"
```

Comment:
Not sure if this is still needed; it was added in https://github.com/moby/moby/pull/5600 to address https://github.com/moby/moby/issues/5173